### ISSUE TITLE
[FEATURE REQUEST] Add published holds to procedures

### ISSUE THREAD

**Issue by AlexSkeleton12** [IVAO-US/SectorFiles#4]
**Is your feature request related to a problem? Please describe.**
Nope, just a desire.

**Describe the solution you'd like**
Add published holds to SIDs and STARs

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by wes-sleeman** [comment]
How do you propose we distinguish required holds (i.e. holds that are a part of the procedure) from published holds that ATC can but doesn't need to use? Do we add them to the holds list and not add them to the procedures? Something else?

**Comment by AlexSkeleton12** [comment]
> How do you propose we distinguish required holds (i.e. holds that are a part of the procedure) from published holds that ATC can but doesn't need to use? Do we add them to the holds list and not add them to the procedures? Something else?

I think ATC should be able to distinguish the difference between a published hold and a hold in lieu of procedure turn. Otherwise, you could probably add them as holds but not each individual hold, but all of the holds for that specific procedure. Ex. I click on ANJLL4 as a hold and all of the holds for that procedure are enabled.

**Comment by wes-sleeman** [comment]
> I think ATC should be able to distinguish the difference between a published hold and a hold in lieu of procedure turn.  

I agree that an experienced controller can, but can a newly minted ADC?

> add them as holds but not each individual hold, but all of the holds for that specific procedure  

Ah, that's a really good idea, actually. I'll see if I can find a data source for published holds and, if I can, I'll incorporate them in this fashion.

**Comment by wes-sleeman** [comment]
Opening this up with a `help wanted` tag. If anyone finds a good data source that's free, public, official, and maintained I'll add it.
